CHANGELOG.md

```diff
@@ -3,6 +3,141 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.11.0
+
+_May 12, 2022_
+
+We'd like to offer a big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Premium plan release
+
+  We’re happy to announce that the Premium plan is [finally out](https://mui.com/pricing/)!
+  With it, MUI X officially steps up to the next level, supporting the most advanced use cases for UI components.
+
+  This plan is available by using the new package `@mui/x-data-grid-premium` which is now containing the row grouping end the excel export.
+
+  If you were already using the row grouping feature, you can upgrade by installing `@mui/x-data-grid-premium`, replace `DataGridPro` by `DataGridPremium` as follow.
+  Notice that the experimental flag is not required anymore
+
+  ```diff
+  -import { DataGridPro } from '@mui/x-data-grid-pro'
+  +import { DataGridPremium } from '@mui/x-data-grid-premium'
+
+  -<DataGridPro experimentalFeatures={{ rowGrouping: true }} />
+  +<DataGridPremium />
+  ```
+
+  For more informations about the new pricing model, please have a look at the blog post.
+
+- 👔 Add Excel export
+
+- 🔎 Quick filtering
+
+  You can now add a quick filtering search bar to your grid.
+  To do so, either pass `showQuickFilter` prop to the `<GridToolbar />` or use the `<GridToolbarQuickFilter />` component in your custom toolbar.
+
+  More information about how to customize the filtering logic in the [documentation](https://mui.com/x/react-data-grid/filtering/#quick-filter)
+
+  ![image](https://user-images.githubusercontent.com/45398769/167364837-6d7b232c-50c9-49a9-9366-26c6d2952716.png)
```

Comment:
The image is a bit blurry, this one should look better
<img width="724" alt="Screenshot 2022-05-10 at 20 43 18" src="https://user-images.githubusercontent.com/13808724/167700105-5a5acc7c-5463-4871-8514-3d09e2f365ae.png">